services/webhook/notifier.go

```diff
@@ -941,3 +944,114 @@ func notifyPackage(ctx context.Context, sender *user_model.User, pd *packages_mo
 		log.Error("PrepareWebhooks: %v", err)
 	}
 }
+
+func (*webhookNotifier) WorkflowJobStatusUpdate(ctx context.Context, repo *repo_model.Repository, sender *user_model.User, job *actions_model.ActionRunJob, task *actions_model.ActionTask) {
```

Comment:
Can this webhook be customized from the UI?

Reply:
Yes and more. So that users cannot cancel to receive this kind of webhook messages. Maybe some people want to do that. I leant from commit status webhook that many people want to avoid to receive that webhook.